extensions/ql-vscode/src/local-queries/skeleton-query-wizard.ts

```diff
@@ -253,24 +252,12 @@ export class SkeletonQueryWizard {
       return undefined;
     }
 
-    const qlPack = load(await readFile(qlPackPath, "utf8")) as
```

Comment:
Ah, I didn't realise we already had this code for manually parsing a query pack. However I think we should still hold ourselves to high standards even though it's pre-existing code, because we're now using it in a new location that will get a lot more usage and run on all query packs in the workspace.